bedrock/firefox/tests/test_base.py

```diff
@@ -498,326 +498,6 @@ def test_fx_default_whatsnew(self, render_mock):
-        req.locale = "en-US"
-        self.view(req, version="123.0")
-        template = render_mock.call_args[0][1]
-        assert template == ["firefox/whatsnew/whatsnew-fx123-eu.html"]
-
-    @override_settings(DEV=True)
-    def test_fx_123_0_0_de(self, render_mock):
-        """Should use whatsnew-fx123-eu template for de locale"""
-        req = self.rf.get("/firefox/whatsnew/")
-        req.locale = "de"
-        self.view(req, version="123.0")
-        template = render_mock.call_args[0][1]
-        assert template == ["firefox/whatsnew/whatsnew-fx123-eu.html"]
-
-    @override_settings(DEV=True)
-    def test_fx_123_0_0_fr(self, render_mock):
-        """Should use whatsnew-fx123-eu template for fr locale"""
-        req = self.rf.get("/firefox/whatsnew/")
-        req.locale = "fr"
-        self.view(req, version="123.0")
-        template = render_mock.call_args[0][1]
-        assert template == ["firefox/whatsnew/whatsnew-fx123-eu.html"]
-
-    @override_settings(DEV=True)
-    def test_fx_123_0_0_us(self, render_mock):
-        """Should use whatsnew-fx123-na template for en-US locale"""
-        req = self.rf.get("/firefox/whatsnew/")
-        req.locale = "en-US"
-        self.view(req, version="123.0")
-        template = render_mock.call_args[0][1]
-        assert template == ["firefox/whatsnew/whatsnew-fx123-na.html"]
-
-    # end 123.0 whatsnew tests
-
-    # begin 124.0 whatsnew tests
-
-    @override_settings(DEV=True)
-    def test_fx_124_0_0_fr(self, render_mock):
-        """Should use index template for fr locale"""
-        req = self.rf.get("/firefox/whatsnew/")
-        req.locale = "fr"
-        self.view(req, version="124.0")
-        template = render_mock.call_args[0][1]
-        assert template == ["firefox/whatsnew/index.html"]
-
-    @override_settings(DEV=True)
-    def test_fx_124_0_0_pl(self, render_mock):
-        """Should use index template for pl locale"""
-        req = self.rf.get("/firefox/whatsnew/")
-        req.locale = "pl"
-        self.view(req, version="124.0")
-        template = render_mock.call_args[0][1]
-        assert template == ["firefox/whatsnew/index.html"]
-
-    @override_settings(DEV=True)
-    def test_fx_124_0_0_de(self, render_mock):
-        """Should use index template for de locale"""
-        req = self.rf.get("/firefox/whatsnew/")
-        req.locale = "de"
-        self.view(req, version="124.0")
-        template = render_mock.call_args[0][1]
-        assert template == ["firefox/whatsnew/index.html"]
-
-    @override_settings(DEV=True)
-    def test_fx_124_0_0_us(self, render_mock):
-        """Should use whatsnew-fx124-na-pdf template for en-US locale"""
-        req = self.rf.get("/firefox/whatsnew/")
-        req.locale = "en-US"
-        self.view(req, version="124.0")
-        template = render_mock.call_args[0][1]
-        assert template == ["firefox/whatsnew/whatsnew-fx124-na-pdf.html"]
-
-    # end 124.0 whatsnew tests
 
     # begin 125.0 whatsnew tests
```

Comment:
I think this is what `ruff` [dislikes](https://github.com/mozilla/bedrock/actions/runs/9472271392/job/26097370005?pr=14670#step:4:6050) — there will be two consecutive empty lines. I guess if you just remove the additional new line, it would pass.